src/scene/vcModel.cpp

```diff
@@ -474,16 +476,53 @@ void vcModel::HandleContextMenu(vcState *pProgramState)
 
   ImGui::Separator();
 
+#if VC_HASCONVERT
   if (((m_pPreferredProjection == nullptr && pProgramState->gis.SRID == 0) || (m_pPreferredProjection != nullptr && m_pPreferredProjection->srid == pProgramState->gis.SRID)) && (m_defaultMatrix == m_sceneMatrix))
   {
-    // Reenable in future
-    //if (ImGui::Selectable(vcString::Get("sceneExplorerExportLAS"), false))
-    //{
-    //  vdkPointCloud_Export(m_pPointCloud, "Testing.las", nullptr);
-    //}
+    if (ImGui::BeginMenu(vcString::Get("sceneExplorerExportPointCloud")))
+    {
+      static vcQueryNode *s_pQuery = nullptr;
+
+      if (ImGui::IsWindowAppearing())
+        s_pQuery = nullptr;
+
+      if (ImGui::BeginCombo(vcString::Get("sceneExplorerExportQueryFilter"), (s_pQuery == nullptr ? vcString::Get("sceneExplorerExportEntireModel") : s_pQuery->m_pNode->pName)))
+      {
+        if (ImGui::MenuItem(vcString::Get("sceneExplorerExportEntireModel"), nullptr, (s_pQuery == nullptr)))
+          s_pQuery = nullptr;
+
+        ContextMenuListModels(pProgramState, pProgramState->activeProject.pFolder->m_pNode, (vcSceneItem**)&s_pQuery, "QFilter", true);
+        ImGui::EndCombo();
+      }
+
+      vcIGSW_FilePicker(pProgramState, vcString::Get("sceneExplorerExportFilename"), pProgramState->modelPath, SupportedTileTypes_QueryExport, vcFDT_SaveFile, nullptr);
+
+      if (ImGui::Button(vcString::Get("sceneExplorerExportBegin")))
+      {
+        if (udFileExists(pProgramState->modelPath) != udR_Success || vcModals_OverwriteExistingFile(pProgramState->modelPath))
+        {
+          vdkQueryFilter *pFilter = ((s_pQuery == nullptr) ? nullptr : s_pQuery->m_pFilter);
+          vdkPointCloud *pCloud = m_pPointCloud;
```

Comment:
Is this required for the capture? I would have thought you could just use `m_pPointCloud` directly? Also, if you remove the model from the scene during an export, does this trigger a crash?